src/sort_manager.hpp

```diff
@@ -25,9 +25,17 @@
 #define _HAS_STD_BYTE 0
 #define NOMINMAX
 
+#ifdef __APPLE__
+// std::filesystem is not supported on Mojave
 #include "../lib/include/filesystem.hpp"
-
 namespace fs = ghc::filesystem;
+#else
+#if __has_include(<filesystem>)
+#include <filesystem>
+#else
+#include <experimental/filesystem>
+#endif
+#endif
```

Comment:
it might be worth putting this in a new header "filesystem.hpp", or something like that

Reply:
moved to chia_filesystem.hpp